WeakAurasTemplates/TriggerTemplates.lua

```diff
@@ -962,7 +1007,16 @@ function WeakAuras.CreateTemplateView(frame)
       end
       button:SetFullWidth(true);
       button:SetClick(function()
-        if (newView.existingAura) then
+        if newView.batchStep then
+          for x, b in pairs(subTypesButtons) do
+            if (x == k) then
+              b.frame:LockHighlight();
+            else
+              b.frame:UnlockHighlight();
+            end
+          end
+          newView.choosenItemBatchSubType[item.spell] = subType;
```

Comment:
Index by item, not by item.spell. We can have multiple items with the same spell.

Reply:
good point